src/bridge/cockpitinternalmetrics.c

```diff
@@ -527,9 +518,6 @@ cockpit_internal_metrics_finalize (GObject *object)
 {
   CockpitInternalMetrics *self = COCKPIT_INTERNAL_METRICS (object);
 
-  g_free (self->instances);
-  g_free (self->omit_instances);
```

Comment:
This removal is incorrect.  I'll put that back.

Reply:
Nice to see that the valgrind tests caught this too!

Fixed in the latest push.